docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,170 @@ description: Next.js supports built-in image optimization, as well as third part
-
-```js
-module.exports = {
-  images: {
-    domains: ['example.com'],
-  },
-}
-```
+### Domains
 
-### Loader
+Sometimes you may want to access a remote image, but still use the built-in Next.js Image Optimization API. To do this, simply leave the `loader` at it's default setting and enter an absolute url for the Image `src`.
 
-If you want to use a cloud provider to optimize images instead of using the Next.js' built-in Image Optimization, you can configure the loader and path prefix. This allows you to use relative urls for the Image `src` and automatically generate the correct absolute url for your provider.
+To protect your application from abuse, you must define a list of remote domains that you intend to access this way. This is done in your `next.config.js` file, as shown below:
 
 ```js
 module.exports = {
   images: {
-    loader: 'imgix',
-    path: 'https://example.com/myaccount/',
+    domains: ['example.com', 'example2.com'],
   },
 }
 ```
 
-The following Image Optimization cloud providers are included:
-
-- [Vercel](https://vercel.com): Works automatically when you deploy on Vercel, no configuration necessary. [Learn more](https://vercel.com/docs/next.js/image-optimization)
-- [Imgix](https://www.imgix.com): `loader: 'imgix'`
-- [Cloudinary](https://cloudinary.com): `loader: 'cloudinary'`
-- [Akamai](https://www.akamai.com): `loader: 'akamai'`
-- Custom: `loader: 'custom'` use a custom cloud provider by implementing the [`loader`](/docs/api-reference/next/image.md#loader) prop on the `next/image` component
-- Default: Works automatically with `next dev`, `next start`, or a custom server
-
-If you need a different provider, you can use the [`loader`](/docs/api-reference/next/image.md#loader) prop with `next/image`.
-
-> The `next/image` component's default loader is not supported when using [`next export`](/docs/advanced-features/static-html-export.md). However, other loader options will work.
-
-> The `next/image` component's default loader uses [`squoosh`](https://www.npmjs.com/package/@squoosh/lib) because it is quick to install and suitable for a development environment. When using `next start` in your production environment, it is strongly recommended that you install [`sharp`](https://www.npmjs.com/package/sharp) by running `yarn add sharp` in your project directory. This is not necessary for Vercel deployments, as `sharp` is installed automatically.
+## Image Sizing
 
-## Caching
+One of the ways that images most commonly hurt performance is through *layout shift*, where the image pushes other elements around on the page as it loads in. This performance problem is so annoying to users that it has it's own Core Web Vital, called [Cumulative Layout Shift](https://web.dev/cls/).  The way to avoid image-based layout shifts is to [always size your images](https://web.dev/optimize-cls/#images-without-dimensions). This allows the browser to reserve precisely enough space for the image before it loads.
```

Comment:
```suggestion
One of the ways that images most commonly hurt performance is through *layout shift*, where the image pushes other elements around on the page as it loads in. This performance problem is so annoying to users that it has its own Core Web Vital, called [Cumulative Layout Shift](https://web.dev/cls/).  The way to avoid image-based layout shifts is to [always size your images](https://web.dev/optimize-cls/#images-without-dimensions). This allows the browser to reserve precisely enough space for the image before it loads.
```